Content.Client/Verbs/VerbSystem.cs

```diff
@@ -178,14 +178,15 @@ public bool TryGetEntityMenuEntities(MapCoordinates targetPos, [NotNullWhen(true
         ///     Ask the server to send back a list of server-side verbs, and for now return an incomplete list of verbs
         ///     (only those defined locally).
         /// </summary>
-        public Dictionary<VerbType, SortedSet<Verb>> GetVerbs(EntityUid target, EntityUid user, VerbType verbTypes)
+        public Dictionary<VerbType, SortedSet<Verb>> GetVerbs(EntityUid target, EntityUid user, VerbType verbTypes,
+            bool force = false)
         {
             if (!target.IsClientSide())
             {
                 RaiseNetworkEvent(new RequestServerVerbsEvent(target, verbTypes));
             }
 
-            return GetLocalVerbs(target, user, verbTypes);
+            return GetLocalVerbs(target, user, verbTypes, force);
```

Comment:
If you want a client to be able to force verbs (basically just ignore can-interact & in-range checks), then you need to somehow communicate that with the server. The way it is done currently, if there are verbs defined in shared systems, the client will think it can execute the verb, but the server wouldn't allow it, so the verb will show up in the menu but do nothing.

I guess you need to add an argument to the `RequestServerVerbsEvent`, and then check if the requesting client satisfies some sorta permission requirement, probably just something like `CanCommand("invokeverb")`

Reply:
Added a check for AdminFlags.Admin. Are there any shared verbs that I can test?

Reply:
This is mostly for admin verbs though, which aren't usually predicted so I'm not sure if this is needed.

Reply:
Yeah I'm also not sure if its really needed for this specific use. Other than admin/debug stuff, there really aren't many verbs related to directly targeting other players. It seems to work though. The "pull" verb is in the menu despite a player being out of range, though the actually pulling system does its own range-checks, and the verb still does nothing.

But I guess its good to have the option for admins to force-verbs in general, so that they can use some verbs on entities without needing to be in range or whatever.